core/modules/ccontrol/testAntlr4GeneratedIR.cc

```diff
@@ -509,7 +509,7 @@ static const vector<Antlr4TestQueries> ANTLR4_TEST_QUERIES = {
             -1
         );},
         "SELECT sce.filterId,sce.filterName "
-            "FROM Science_Ccd_Exposure AS sce "
+            "FROM Science_Ccd_Exposure AS `sce` "
```

Comment:
This is the "correct" string for the test, IE the object built starting on line 470 is expected to create sql statement in this form, with the quotes. I quote all the aliases now because sometimes it is required and (AFAIK) it never hurts. For example, the alias in ```SELECT table.column AS `table.column` ```

Reply:
^ that alias is what would be auto generated for that element if there was no alias specified by the user.